topomodelx/nn/combinatorial/hmc.py

```diff
@@ -72,7 +69,7 @@ def forward(
         neighborhood_2_to_2,
         neighborhood_0_to_1,
         neighborhood_1_to_2,
-    ) -> torch.Tensor:
+    ):
```

Comment:
Please don't just remove the return type but change it appropriately.

Reply:
Adapted.